src/pages/product-updates/index.tsx

```
@@ -9,11 +9,14 @@ import NextSeoWrapper from '@/components/NextSeoWrapper';
 import PageContainer from '@/components/PageContainer';
 import LocalizationMessage from '@/components/Sanity/LocalizationMessage';
 import Page from '@/components/Sanity/Page';
+import { getProductUpdatesPage } from '@/components/Sanity/utils';
```

ESLint

[error] 'getProductUpdatesPage' is defined but never used.

(@typescript-eslint/no-unused-vars)